src/mol-repr/structure/visual/util/link.ts

```diff
@@ -80,7 +81,7 @@ export function calculateShiftDir(out: Vec3, v1: Vec3, v2: Vec3, v3: Vec3 | null
 
 export interface LinkBuilderProps {
     linkCount: number
-    position: (posA: Vec3, posB: Vec3, edgeIndex: number) => void
+    position: (posA: Vec3, posB: Vec3, edgeIndex: number, adjust: boolean) => void
```

Comment:
This function is used in quite a few files. Does your current solution compile?

![image](https://github.com/molstar/molstar/assets/21145776/ac106ec1-2aae-4bbc-afe3-9e39f88a7b36)

Reply:
For me `npm run build` succeeds and the viewer works. In JavaScript all those instances are allowed to ignore trailing arguments, making this a fairly lightweight change.